.circleci/config.yml

```diff
@@ -128,8 +141,9 @@ commands:
         type: boolean
         default: false
       env:
-        type: string
-        default: local
+        type: enum
+        enum: [ "local", "staging", "alpha" ]
+        default: "local"
```

Comment:
I've come to avoid quotes unless necessary, since they add a little noise.

Reply:
yeah, quotes are removed.